StitchCore/StitchCoreTests/Admin/AppEndpoint.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import PromiseKit
+@testable import StitchCore
+
+struct AppView: Codable {
```

Comment:
Given this is in a different module, I'm okay with `App`. If it were in StitchCore, I think it would be too confusing. To play devil's advocate against that though, I wouldn't put it past a user to think `App` has methods on it like delete or addService whereas `AppModel` would work.

Fun hack and possible abuse idea:
`struct Model {}`
`extension Model { struct App {} }`

:)